<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1922</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1928</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/camera_bg.png);</string>
   </property>
   <widget class="QLabel" name="stream">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1500</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/newPrefix/imgPlaceholder.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="strip">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>30</y>
      <width>333</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/newPrefix/stripPlaceholder.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img1">
    <property name="geometry">
     <rect>
      <x>1578</x>
      <y>110</y>
      <width>297</width>
      <height>203</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/transparent.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/newPrefix/transparent.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img2">
    <property name="geometry">
     <rect>
      <x>1578</x>
      <y>332</y>
      <width>300</width>
      <height>203</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/transparent.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/newPrefix/transparent.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img3">
    <property name="geometry">
     <rect>
      <x>1578</x>
      <y>556</y>
      <width>297</width>
      <height>203</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/transparent.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/newPrefix/transparent.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fullscreenlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/transparent.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
